tests/foreman/ui/test_discoveredhost.py

```diff
@@ -141,6 +135,234 @@ def setUpClass(cls):
         cls.discovery_auto.value = 'True'
         cls.discovery_auto.update({'value'})
 
+        # Create a new Life-Cycle environment
```

Comment:
to be honest, like to handle in current PR, but I think we can live with another issue too =)
Thanks @sghai